.ci/Jenkinsfile

```diff
@@ -143,6 +144,8 @@ pipeline {
                 post {
                   always {
                     archiveArtifacts(allowEmptyArchive: true, artifacts: "${BASE_DIR}/apm-lambda-extension/.regions")
+                    archiveArtifacts(allowEmptyArchive: true, artifacts: "${BASE_DIR}/apm-lambda-extension/*${SUFFIX_ARN_FILE}")
+                    archiveArtifacts(allowEmptyArchive: true, artifacts: "${BASE_DIR}/apm-lambda-extension/.aws")
```

Comment:
This folder contains the output from the aws command `publish-layer-version`

As far as I see https://docs.aws.amazon.com/cli/latest/reference/lambda/publish-layer-version.html#output does not mention about any sensitive data in the output

![image](https://user-images.githubusercontent.com/2871786/152164914-c63cd842-d39a-4caf-af0b-3f598923daf3.png)


Storing those files will help to debug what's going on if something bad happened while publishing a new release.